src/Homie.hpp

```diff
@@ -37,6 +37,11 @@ class HomieClass {
   void __setFirmware(const char* name, const char* version);
   void __setBrand(const char* brand) const;
 
+  static const BootMode MODE_UNDEFINED = BootMode::UNDEFINED;
```

Comment:
For it to be consistent, can you please make the `BootMode` a `HomieBootMode` and include it the same way it works for https://github.com/marvinroger/homie-esp8266/blob/develop/src/HomieEvent.hpp? This way, the user could use `HomieBootMode::NORMAL` from his code.